tests/requirements.txt

```diff
@@ -1,12 +1,14 @@
+--only-binary=netCDF4
```

Comment:
with `--only-binary=:all:` it fails as `coveralls` isn't publishing binaries in their latest releases :shrug: 

Could also be `--only-binary=:all:` combined with `--no-binary=coveralls`, I think. But it might get tricky to have to keep fixing other dependencies and transitive dependencies. For now I think marking just netcdf4 as binary/wheel should be fine?